### ISSUE TITLE
Please check the bug....

### ISSUE THREAD

**Issue by theodoruszq** [bkong999/COVNet#12]
Thank for sharing the code.

https://github.com/bkong999/COVNet/blob/719d51681f439eb8ab2638c093ef8d40fa32e4c1/main.py#L197

Please make sure the metric is evaluated on validation set instead of on training set in your paper. I am wondering when the bug appears.

**Comment by bkong999** [comment]
> Thank for sharing the code.
> 
> https://github.com/bkong999/COVNet/blob/719d51681f439eb8ab2638c093ef8d40fa32e4c1/main.py#L197
> 
> Please make sure the metric is evaluated on validation set instead of on training set in your paper. I am wondering when the bug appears.

Thanks for reporting this issue. Resolved.
